scripts/lib/svg-generation-fns.ts

```diff
@@ -132,7 +132,9 @@ export async function processSvg(symbolsSvg: string, fileName?: string) {
           }
 
           // Check if there's a source file for this symbol
-          const outputPath = `./symbols/${groupId}_horz.ts`
+          const isDiode = groupId.toLowerCase().includes("diode")
+          const defaultOrientation = isDiode ? "right" : "horz"
+          const outputPath = `./symbols/${groupId}_${defaultOrientation}.ts`
```

Comment:
this code is messy, it was kind of messy before but we shouldn't make it worse

instead of having a default orientation, determine all the orientations in advance.

```ts
const orientations : ["horz", "vert"] | ["left", "up", "right", "down"] = /*...*/
```